OpenRA.Mods.Common/UtilityCommands/ExtractZeroBraneStudioLuaAPI.cs

```diff
@@ -0,0 +1,126 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2016 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using System.Reflection;
+using OpenRA.Scripting;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	// See https://studio.zerobrane.com/doc-api-auto-complete for reference
+	class ExtractZeroBraneStudioLuaAPI : IUtilityCommand
+	{
+		public string Name { get { return "--zbstudio-lua-api"; } }
+
+		public bool ValidateArguments(string[] args)
+		{
+			return true;
+		}
+
+		[Desc("Generate a ZeroBrane Studio Lua API and auto-complete description.")]
```

Comment:
Seeing as you are outputting to the stdout the user must redirect the data, it would be polite to state that it should be written to a lua file.

Currently this is a guessing game (or the person has to stumble across https://github.com/pkulchenko/ZeroBranePackage/blob/226dbc946624be2a933464d23b4767adb4815d50/openra.lua).

Even though this will probably only be used by us to generate docs for release it doesn't hurt to add helpful information.